### PR TITLE
Updates events-team registration handbook

### PR DESCRIPTION
**Pull request by palnabarun**
This updates the events team registration handbook with the latest questions asked during registration.

/sig contributor-experience
/area contributor-summit
/area na-summit

### CODE REVIEW

File: events/events-team/registration/README.md [hunk 1]
```diff
@@ -60,19 +60,46 @@ workshop organizers.
...
-| If you had a speaking slot, what would you like to talk about? | No       |                                                               |
-
+| Question                                                       | Type      | Required | Notes                                                         |
+|----------------------------------------------------------------|-----------|----------|---------------------------------------------------------------|
+| Name                                                           | Free Text | Yes      |                                                               |
+| Email                                                          | Free Text | Yes      |                                                               |
+| GitHub Handle                                                  | Free Text | Yes      |                                                               |
+| Signed the CLA                                                 | Radio     | Yes      | Provide link to CLA. Options: Yes, No - but will do before workshop, No - cannot sign due to work resrictions. |
```
[the remark this answers is not in the document]
Comment (palnabarun) [reply events/events-team/registration/README.md]: Done. :+1:

File: events/events-team/registration/README.md [hunk 2]
```diff
@@ -60,19 +60,46 @@ workshop organizers.
...
+| Food Restrictions                                              | Dropdown  | Yes      | Options: `none`, `Gluten Free`, `Vegetarian`, `Vegan`, `Other` |
+| Swag Related Question                                          | Dropdown  | Yes      | Swag question is generally shirt size.                        |
+| Social Event                                                   | Dropdown  | Yes      | Required if there is an independent social event.             |
+| Emergency Contact                                              | Free Text | No       | Name and Phone Number. Phone number to be verified.           |
+| How can we make this a valuable event for you?                 | Free Text | No       |                                                               |
+| If you had a speaking slot, what would you like to talk about? | Free Text | No       |                                                               |
+| Where did you hear about the Contributor Summit?               | Free Text | No       | Options: `Twitter`, `mailing list`, `GitHub`, `community meeting`, `SIG/WG Meeting`, `Slack`, `Other` |
```
[the remark this answers is not in the document]
Comment (palnabarun) [reply events/events-team/registration/README.md]: Done :+1:

File: events/events-team/registration/README.md [hunk 3]
```diff
@@ -60,19 +60,46 @@ workshop organizers.
...
+
+Following are some of the path specific questions that have been asked in the past:
+
+__New Contributor__
+
+| Question                                                       | Type      | Required | Notes                                                         |
+|----------------------------------------------------------------|-----------|----------|---------------------------------------------------------------|
+| Laptop availability for workshop                               | Radio     | Yes      | Should be mentioned that workshop can still be attended if laptop is not available and pair with other participants. Options: Yes, No - do not have a laptop, No - have one but cannot install software on it without IT approval |
```
[the remark this answers is not in the document]
Comment (palnabarun) [reply events/events-team/registration/README.md]: Done :+1:

File: events/events-team/registration/README.md [hunk 4]
```diff
@@ -60,19 +60,46 @@ workshop organizers.
...
+| Have you joined the kubernetes slack?                          | Free Text | No       |                                                               |
```
[the remark this answers is not in the document]
Comment (palnabarun) [reply events/events-team/registration/README.md]: Done :+1:

File: events/events-team/registration/README.md [hunk 5]
```diff
@@ -60,19 +60,46 @@ workshop organizers.
...
+| Swag Related Question                                          | Dropdown  | Yes      | Swag question is generally shirt size.                        |
+| Social Event                                                   | Dropdown  | Yes      | Required if there is an independent social event.             |
+| Emergency Contact                                              | Free Text | No       | Name and Phone Number. Phone number to be verified.           |
+| How can we make this a valuable event for you?                 | Free Text | No       |                                                               |
+| If you had a speaking slot, what would you like to talk about? | Free Text | No       |                                                               |
+| Where did you hear about the Contributor Summit?               | Free Text | No       | Options: `Twitter`, `mailing list`, `GitHub`, `community meeting`, `SIG/WG Meeting`, `Slack`, `Other` |
+
+Following are some of the path specific questions that have been asked in the past:
+
+__New Contributor__
```
[the remark this answers is not in the document]
Comment (palnabarun) [reply events/events-team/registration/README.md]: My bad. Forgot about that. :sweat_smile:

File: events/events-team/registration/README.md [hunk 6]
```diff
@@ -60,19 +60,46 @@ workshop organizers.
...
+| Kubernetes areas already contributing to                       | Checkbox  | No       | Options: `API`, `CLI`, `testing`, `docs`, `community`, `release`, etc |
+| Are there any SIG's you are interested in joining?             | Free Text | No       |                                                               |
+| What are you hoping to get out of the contributor summit?      | Free Text | No       |                                                               |
+| Kubernetes areas interested in contributing to                 | Checkbox  | No       | Options: `API`, `CLI`, `testing`, `docs`, `community`, `release`, etc |
+| Experience with Go                                             | Free Text | No       | Rate from 1 to 5, with 1 being no experience                  |
+| Experience with kubernetes as a user                           | Free Text | No       | Rate from 1 to 5, with 1 being no experience                  |
+| Experience of developing application that integrate with kubernetes | Free Text | No  | Rate from 1 to 5, with 1 being no experience                  |
+
+__Active Contributor__
```
[the remark this answers is not in the document]
Comment (palnabarun) [reply events/events-team/registration/README.md]: My bad. Forgot about that. :sweat_smile:

File: events/events-team/registration/README.md [hunk 7]
```diff
@@ -60,19 +60,46 @@ workshop organizers.
...
+| Have you joined the kubernetes slack?                          | Free Text | No       |                                                               |
+| If you had a speaking slot, what would you like to talk about? | Free Text | No       |                                                               |
+| Kubernetes areas already contributing to                       | Checkbox  | No       | Options: `API`, `CLI`, `testing`, `docs`, `community`, `release`, etc |
+| Are there any SIG's you are interested in joining?             | Free Text | No       |                                                               |
+| What are you hoping to get out of the contributor summit?      | Free Text | No       |                                                               |
+| Kubernetes areas interested in contributing to                 | Checkbox  | No       | Options: `API`, `CLI`, `testing`, `docs`, `community`, `release`, etc |
+| Experience with Go                                             | Free Text | No       | Rate from 1 to 5, with 1 being no experience                  |
```
[the remark this answers is not in the document]
Comment (palnabarun) [reply events/events-team/registration/README.md]: Done :+1:

File: events/events-team/registration/README.md [hunk 8]
```diff
@@ -60,19 +60,46 @@ workshop organizers.
...
+| Have you joined the kubernetes slack?                          | Free Text | No       |                                                               |
+| If you had a speaking slot, what would you like to talk about? | Free Text | No       |                                                               |
+| Kubernetes areas already contributing to                       | Checkbox  | No       | Options: `API`, `CLI`, `testing`, `docs`, `community`, `release`, etc |
+| Are there any SIG's you are interested in joining?             | Free Text | No       |                                                               |
+| What are you hoping to get out of the contributor summit?      | Free Text | No       |                                                               |
+| Kubernetes areas interested in contributing to                 | Checkbox  | No       | Options: `API`, `CLI`, `testing`, `docs`, `community`, `release`, etc |
```
[the remark this answers is not in the document]
Comment (palnabarun) [reply events/events-team/registration/README.md]: Done :+1:

File: events/events-team/registration/README.md [hunk 9]
```diff
@@ -60,19 +60,46 @@ workshop organizers.
...
+
+__New Contributor__
+
+| Question                                                       | Type      | Required | Notes                                                         |
+|----------------------------------------------------------------|-----------|----------|---------------------------------------------------------------|
+| Laptop availability for workshop                               | Radio     | Yes      | Should be mentioned that workshop can still be attended if laptop is not available and pair with other participants. Options: Yes, No - do not have a laptop, No - have one but cannot install software on it without IT approval |
+| Have you joined the kubernetes slack?                          | Free Text | No       |                                                               |
+| If you had a speaking slot, what would you like to talk about? | Free Text | No       |                                                               |
+| Kubernetes areas already contributing to                       | Checkbox  | No       | Options: `API`, `CLI`, `testing`, `docs`, `community`, `release`, etc |
```
[the remark this answers is not in the document]
Comment (palnabarun) [reply events/events-team/registration/README.md]: Thanks for the idea behind the question. I had missed it. This is a nice metric to have.

**Comment by mrbobbytables** [comment]
woops -- let this slide by for a few days 😬 sorry about that.

Thanks for adding the questions!

/lgtm
/approve

**Comment by palnabarun** [comment]
Thank you @mrbobbytables for merging this. Happy to help. :smile:

**Pull request merged**